app/templates/home.html

```diff
@@ -6,4 +6,15 @@
     <h1>{{message}}</h1>
 </div>
 
-{% endblock %}
\ No newline at end of file
+<!-- I chose to add the quote day to the home page until the relavent calendar view will be developed.  -->
```

Comment:
i would suggest adding a demo html for adding it because in the future the daily quote might not be there.

Reply:
Thanks for the comment!
Since its just a simple sentence, which changes every day, I see no reason to upload a demo page. I think it will eventually appear in the main view of the calendar, but anyway I'll follow up on it and make sure to move it accordingly...